macros/edr/system/hooks/on_run_end.sql

```diff
@@ -0,0 +1,22 @@
+{% macro on_run_end() %}
```

Comment:
Add debug logs for timings?

Reply:
I'm thinking of adding logs to the inner macros - `upload_dbt_invocation`, `upload_dbt_artifacts`, `upload_run_results`, etc so that the users would understand what happens at which step. What do you think?